### PR TITLE
Mark non-npm packages

### PR DESCRIPTION
**Pull request by sandersn**
Mark non-npm packages as such.

Notes:

* Node is marked as non-npm because it is, in fact, not on npm.
* jsqrcode has a node equivalent, but the version on DT is the non-npm version.
* titanium actually *is* on npm, but the version on DT is actually titanium_mobile which is *not* on npm.
* I moved shipit to shipit-cli (which is what the typings are actually for), so that at least it matches the npm name.

### CODE REVIEW

**Review by zspitz — APPROVED** [review]

**Review by taoqf — APPROVED** [review]

File: types/ember__application/index.d.ts [hunk 1]
```diff
@@ -1,4 +1,4 @@
-// Type definitions for @ember/application 3.0
+// Type definitions for non-npm package @ember/application 3.0
```
Comment (mike-north) [inline types/ember__application/index.d.ts]: I'm not sure how the `@ember/*` packages should be handled. They're published to NPM (combined) as [ember-source](https://www.npmjs.com/package/ember-source), but consumed by users as individual `@ember/*` modules.

**Review by akrantz — APPROVED** [review]

**Review by agasbzj — APPROVED** [review]
Approve for amap-js-sdk, thank you.

File: types/ember__application/index.d.ts (hunk 1 above)
Comment (sandersn) [reply types/ember__application/index.d.ts]: 1. This is a machine-readable marker first, human-readable second, and it's *technically* true that `@ember/application` isn't on npm, so I think this should be here. Maybe I should allow an alternate wording, however.
2. What do the import statements look like? Are they literally `import { x } from '@ember/application'`?

Comment (mike-north) [reply types/ember__application/index.d.ts]: > Are they literally `import { x } from '@ember/application'`
yes

**Review by samuelmaddock — APPROVED** [review]

**Review by Quas94 — APPROVED** [review]

**Review by Fluccioni — APPROVED** [review]

**Review by johngeorgewright — APPROVED** [review]

**Review by diagramatics — APPROVED** [review]

**Review by pluma — APPROVED** [review]

**Review by ghost — APPROVED** [review]

**Review by carrrywu — APPROVED** [review]

**Review by skubarenko — APPROVED** [review]

**Review by jafaircl — APPROVED** [review]

**Review by danmarshall — APPROVED** [review]

**Pull request merged**